.github/workflows/ci.yml

```diff
@@ -31,3 +31,37 @@ jobs:
         python -m pip install -e .[bayes,composable,tests]
     - name: Test with unittest
       run: python -m unittest
+
+  # build and push documentation to gh-pages (only if pushed to the master branch)
+  docs:
+    name: Documentation
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build documentation
+      uses: ammaraskar/sphinx-action@master
+      with:
+        pre-build-command: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install -e .[docs]
+          mkdir -p docs/source/wiki/wiki_examples/selected_plots
+          cp docs/source/wiki_editable/wiki_examples/selected_plots/* docs/source/wiki/wiki_examples/selected_plots/
+          find docs/source/wiki_editable -name '*.md' -exec sh -c 'pandoc -f markdown -t rst "$$1" -o "docs/source/wiki/$$(basename "$$1" .md).rst"' _ {} \;
```

Comment:
This line might fail because `pandoc` might not be installed yet. Ideally, you test and debug the CI when merging `devel` into master.

Reply:
This is now fixed by using the MyST plugin within sphinx, through which no separate conversion step with pandoc is needed, any more.